### ISSUE TITLE
some `color` primitives make the conversion fails

### ISSUE THREAD

**Issue by Doublonmousse** [jgm/typst-hs#35]
These are the 
- `desaturate` method, fails with `Color does not have a method "desaturate" or FieldAccess requires a dictionary`
- `color.hsl` constructor, fails with `Identifier "color" not found`
- `color.hsv` constructor, fails with `Identifier "color" not found`
- `color.linear-rgb` constructor, fails with `Identifier "color" not found`
- `.mix` method, fails with `Color does not have a method "mix" or FieldAccess requires a dictionary`
- `oklab` constructor, fails with `Identifier "oklab" not found`
- `oklch` constructor, fails with `Identifier "oklch" not found`
- `rotate` method, fails with `Color does not have a method "rotate" or FieldAccess requires a dictionary`
- `saturate` method fails with `Color does not have a method "saturate" or FieldAccess requires a dictionary`

**Comment by Doublonmousse** [comment]
reproduction : https://github.com/Doublonmousse/pandoc-typst-reproducer/tree/main/color_issues

**Comment by jgm** [comment]
Right.  We don't yet support all of the color methods.  This could be added, but the color support is quite complex and it's not a high priority at the moment.
